### DISCUSSION TITLE
License scanning not returning meaningful results from the server

### DISCUSSION THREAD

**Discussion by cemugo** [aquasecurity/trivy#4950] [Q&A] [answered]
### Question

Is it possible to run the license scanning from docker images on the server side?

Requesting license scanning to the server returns empty results contrary to the stand-alone client.

Thank you very much in advance.

Regards,
César M.

### Target

Container Image

### Scanner

License

### Output Format

JSON

### Mode

Client/Server

### Operating System

client: macOS - server: alpine:3.18.2

### Version

```bash
Version: 0.44.0
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-08-04 00:13:00.997759679 +0000 UTC
  NextUpdate: 2023-08-04 06:13:00.997759379 +0000 UTC
  DownloadedAt: 2023-08-04 05:15:04.477161 +0000 UTC
```

**Comment by nikpivkin** [comment]
Hi @cemugo !

Could you please specify an image or give an example of a Dockerfile and what flags do you use when running Trivy?

**Reply by cemugo** [reply]
Hello @nikpivkin Thank you very much for the fast answer!

I'm testing the scan with the official Trivy image: "aquasec/trivy"

These are the commands:

* stand-alone client: trivy image -f json --severity UNKNOWN,MEDIUM,HIGH,CRITICAL --scanners license aquasec/trivy
* server-client: trivy image -f json --server https://here-server-url --severity UNKNOWN,MEDIUM,HIGH,CRITICAL --scanners license aquasec/trivy

Do you want me to post the results of both executions?

Thank you very much.

Regards,
César M.

**Comment by nikpivkin** [comment, marked as answer]
Thanks @cemugo !

I opened the issue https://github.com/aquasecurity/trivy/issues/4951.

**Reply by cemugo** [reply]
Thanks a million for checking and the lightning-fast answer, @nikpivkin 

Take care and cheers 🍻 

Regards,
César M.

**Comment by bakuljajan** [comment]
Hi @nikpivkin, I've wondering if this issue is fixed yet? I always got Severity and Category `unknown` when using client-server mode. Doing the scanning on standalone mode gives out both records just fine.
![image](https://github.com/user-attachments/assets/5a1c4965-1f40-4e9a-b733-b0d4fc84b84d)

![image](https://github.com/user-attachments/assets/654d776f-dbf8-4af7-8256-c028478aa95b)

**Reply by nikpivkin** [reply]
@bakuljajan Hi! I'll take a look

**Reply by nikpivkin** [reply]
Track https://github.com/aquasecurity/trivy/issues/7202
